xml/cap_admin_configuration_changes.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha.cap.configuration-changes"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Configuration Changes</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <para>
+  After the initial deployment of &cap;, any changes made to your &helm; chart
+  values, whether through your <filename>scf-config-values.yaml</filename> file
+  or directly using the <literal>--set</literal> flag, are applied using the
```

Comment:
The `-set` flag is for kubectl? This should specify which command it's for. Also, commands and command options use the `<command>` tag.

Reply:
Clarified in https://github.com/SUSE/doc-cap/pull/291/commits/1c5ce68bb575f15ee30c6cc5d37666d64bb3745d

Reply:
:+1: